### DISCUSSION TITLE
Multiple login restrictions

### DISCUSSION THREAD

**Discussion by wu-clan** [fastapi-practices/fastapi_best_architecture#71] [Ideas]
For multi-login restrictions, see:

- https://github.com/flipped-aurora/gin-vue-admin/blob/b1510b55e80708b2bb3d08c1a1ade6f845129f18/server/middleware/jwt.go#L65
https://github.com/flipped-aurora/gin-vue-admin/blob/b1510b55e80708b2bb3d08c1a1ade6f845129f18/server/api/v1/system/sys_user.go#L93

**Comment by wu-clan** [comment]
Hi, @downdawn 

What do you have in mind?

**Reply by downdawn** [reply]
Adding to the blacklist or directly clearing each has its own advantages, and we are now directly clearing.

His multi-sign-on here is the same as the single-sign-on I understand. .

**Reply by wu-clan** [reply]
The current approach seems to clean the data better, Thank you for your efforts.
